### PR TITLE
Prysm Flow: fixing readme location

### PR DESCRIPTION
**Pull request by james-prysm**
Realized readme for prysm flows had some problems, wanted to fix them.

### CODE REVIEW

File: flows/client-specific/prysm/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Authentication: Prysm requires a bearer token for using the validator APIs, this token is also used for Prysm's native UI. For more information on where to find this token please look at the Prysm Docs.
+
+2. CORS: Prysm has CORS protection with ports 3000 (typically REACT), 7500, and 4200 (typically Angular). Please use the appropriate Prysm flag to enable other ports. Please look at the Prysm Docs for more information.
+
+3. Prysm Wallet & keymanager Type: Prysm uses something called a Prysm wallet (not related to hardware wallet) to manage the validator keys inside the Prysm client. All users of Prysm must create a wallet with a specific keymanager type in order to import or delete validator keys. The following errors will be thrown if requirements are not met:
+- invalid wallet: either the wallet itself is corrupted, missing, or unable to be opened.
+- invalid keymanager type: the keymanager type when creating the wallet isn't usable with the keymanager APIs. Only the `local` keymanager type is allowed to be used for Keymanager APIs referring to the type where users are storing the validators in their local Prysm instance instead of using a remote signer.
```
Comment (rolfyone) [inline flows/client-specific/prysm/README.md:19]: i thought you were building web3signer support? just curious whether this is an accurate statement if that is the case...

File: flows/client-specific/prysm/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- validator service cannot get keymanager: The Keymanager API is called too soon after wallet creation.
```
Comment (rolfyone) [inline flows/client-specific/prysm/README.md:20]: this is interesting, maybe service startup order would eliminate that?

**Review by rolfyone — APPROVED** [review]
LGTM - just a couple of questions out of curiosity really.

File: flows/client-specific/prysm/README.md (hunk 1 above)
Comment (james-prysm) [reply flows/client-specific/prysm/README.md:19]: you're right, i wrote this at the time for the regular keymanager API before implementing the remote keymanager api🤔  I guess I should specify that in this flow. ( will do that in a separate pr)

File: flows/client-specific/prysm/README.md (hunk 2 above)
Comment (james-prysm) [reply flows/client-specific/prysm/README.md:20]: startup order does matter, a few other apis/ cli commands need to be called as well. and unfortunately requires the beacon node to be synced 😓

**Pull request merged**